kairon/actions/definitions/http.py

```diff
@@ -94,6 +94,8 @@ async def execute(self, dispatcher: CollectingDispatcher, tracker: Tracker, doma
             self.__is_success = True
             slot_values, slot_eval_log = ActionUtility.fill_slots_from_response(http_action_config.get('set_slots', []),
                                                                                 http_response)
+            if dispatch_type == DispatchType.json.value and isinstance(bot_response, str):
```

Comment:
use try except block to handle instances where the http response is string but is not a valid json.